apps/cesium.omniverse.dev.kit

```diff
@@ -17,6 +17,7 @@ app.useFabricSceneDelegate = true
 app.usdrt.scene_delegate.enableProxyCubes = false
 app.usdrt.scene_delegate.geometryStreaming.enabled = false
 app.fastShutdown = true
+app.viewport.grid.enabled = false
```

Comment:
Could we not default this to off? It's easy enough to turn off manually and then it persists between sessions. I typically leave it on because it can be useful in some cases. By setting it off in this file, it turns it off every time for me.

Reply:
Sure, I removed this line